qt/scientific_interfaces/Inelastic/Analysis/IndirectFitPropertyBrowser.h

```diff
@@ -38,10 +38,10 @@ namespace IDA {
 using namespace Mantid::API;
 using namespace MantidWidgets;
 
-class FunctionTemplateBrowser;
-class FitStatusWidget;
+class MANTIDQT_INELASTIC_DLL FunctionTemplateBrowser;
+class MANTIDQT_INELASTIC_DLL FitStatusWidget;
```

Comment:
These are just forward declarations and so will not need the export macro
https://stackoverflow.com/questions/4757565/what-are-forward-declarations-in-c